styles.css

```diff
@@ -52,11 +57,13 @@
 
 #closeButton,
 #continueButton {
+  margin: 0;
+  padding: 0;
```

Comment:
@rezaarkan Let's have just one line for defining `padding` instead of having it in both lines 61 and 64. Thank you!